catalog/imaging/imaging-transformer-nitf/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -13,176 +13,184 @@
  **/
  -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
-           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
-           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+  xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0
            http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
            http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
            http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd">
 
-    <bean id="imageMetacardType" class="org.codice.alliance.transformer.nitf.image.ImageMetacardType">
-    </bean>
-
-    <bean id="imageMetacardFactory" class="org.codice.alliance.transformer.nitf.MetacardFactory">
-        <property name="metacardType" ref="imageMetacardType"/>
-    </bean>
-
-    <bean id="gmtiMetacardType" class="org.codice.alliance.transformer.nitf.gmti.GmtiMetacardType">
+  <bean id="imageMetacardType" class="org.codice.alliance.transformer.nitf.image.ImageMetacardType">
+  </bean>
+
+  <bean id="imageMetacardFactory" class="org.codice.alliance.transformer.nitf.MetacardFactory">
+    <property name="metacardType" ref="imageMetacardType"/>
+  </bean>
+
+  <bean id="nitfUtilies" class="org.codice.alliance.transformer.nitf.NitfUtilities">
```

Comment:
This is all a reformat except this bean and the countryCodeConverter service reference.